modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -239,6 +239,57 @@ The Zipkin receiver ingests data in the Zipkin v1 and v2 formats.
 [id="processors_{context}"]
 === Processors
 
+
+[id="batch-processor_{context}"]
+==== Batch processor
+
+The batch processor batches the data to reduce the number of outgoing connections needed to transfer the telemetry information.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.Example of the OpenTelemetry Collector custom resource when using the Batch processor
+[source,yaml]
+----
+  config: |
+    processor:
+      batch:
+        timeout: 5s
+        send_batch_max_size: 10000
+    service:
+      pipelines:
+        traces:
+          processors: [batch]
+        metrics:
+          processors: [batch]
+----
+
+.Parameters used by the Batch processor
+[cols="3",options="header"]
+|===
+|Parameter |Description |Default
+
+| `timeout`
+| Sends the batch after a specific time duration, irrespective of its size.
+| 200ms
+
+| `send_batch_size`
+| Sends the batch of telemetry data after the specified number of spans or metrics.
+| 8192
+
+| `send_batch_max_size`
+| The maximum allowable size of the batch. Must be equal or greater than `send_batch_size`.
+| 0
```

Comment:
I'm curious--what are the units for this parameter?

Reply:
Number of spans, metric data points, or log records